### PR TITLE
Change commentextensionlist ext element to 2006/main namespace

### LINKED ISSUE #1751: Bug with Open XML SDK
**Describe the bug**
```xml
In part \ppt\comment\modernComment... part is coded in the SDK metadata as p188:ext. This causes OpenXmlUnknownElement to be assigned as the child of p188:extLst.
       <p188:extLst>
            <p:ext xmlns:p="http://schemas.openxmlformats.org/presentationml/2006/main" uri="{5BB2D875-25FF-4072-B9AC-8F64D62656EB}">
                <p228:taskDetails xmlns:p228="http://schemas.microsoft.com/office/powerpoint/2022/08/main">
                    <p228:history>
                        <p228:event time="2024-07-16T05:56:24.928" id="{22A90AAB-29DC-4F45-80B8-41660EB3CE91}">
                            <p228:atrbtn authorId="{DDBD2B34-3E97-38F7-B7A9-5BD8D52D7288}"/>
                            <p228:anchr>
                                <p228:comment id="{C5BB4569-00FC-429B-B44C-282847B06E34}"/>
                            </p228:anchr>
                            <p228:add/>
```

This should be fixed on our schema processor.

**To Reproduce**
Create a presentation with a modern comment. Then inspect the \ppt\comment\modernCommentXXXX.xml part. The above shows p:ext and that is per the Office schemas. 

However, when parsing in SDK code: 

```csharp
    SlidePart sp = pd.PresentationPart.GetPartsOfType<SlidePart>().First();

    PowerPointCommentPart ppcp = sp.GetPartsOfType<PowerPointCommentPart>().First();

    if (ppcp.RootElement is null)
    {
        throw new InvalidDataException("no root element!");
    }

    CommentPropertiesExtensionList cpel = ppcp.RootElement.Descendants<CommentPropertiesExtensionList>().First();

    if (cpel is null)
    {
        throw new InvalidDataException("Presentation modern comment part is missing p188:extlst");
    }

    CommentPropertiesExtension cpe = cpel.ChildElements.First<CommentPropertiesExtension>();
    
    if (cpe is null)
    {
// This line will throw because cpe is null due to the p:ext being wrapped by OpenXmlUnknownElement. 
// In our patched schemas we code this as p188:ext which is wrong. 
        throw new InvalidDataException("Presentation modern comment part is missing p188:ext");
    }
```
**Steps to reproduce the behavior:**
Use the code above to access the ext child element the extLst in a p188:cm block.

**Observed behavior**
`cpe` is null.

**Expected behavior**
In the code above, cpe should be a valid CommentPropertiesExtension object wrapping the p:ext.

**Desktop (please complete the following information):**
 - OS: Windows
 - .NET Target: .Net 8.0
 - DocumentFormat.OpenXml Version: 3.0.2

### PR DESCRIPTION
**Pull request by tomjebo**
- fixes #1751

### CODE REVIEW

File: generated/DocumentFormat.OpenXml.Linq/DocumentFormat.OpenXml.Generator/DocumentFormat.OpenXml.Generator.OpenXmlGenerator/Linq.P188.g.cs [hunk 1]
```diff
@@ -82,27 +82,14 @@ public static class P188
...
-        /// </summary>
-        /// <remarks>
-        /// <para>As an XML element, it:</para>
-        /// <list type="bullet">
-        /// <item><description>has the following parent XML elements: <see cref="extLst" />.</description></item>
-        /// <item><description>has the following child XML elements: <see cref="P223.reactions" />, <see cref="P228.taskDetails" />.</description></item>
-        /// <item><description>corresponds to the following strongly-typed classes: CommentPropertiesExtension.</description></item>
-        /// </list>
-        /// </remarks>
-        public static readonly XName ext = p188 + "ext";
```
Comment (twsouthwick) [inline generated/DocumentFormat.OpenXml.Linq/DocumentFormat.OpenXml.Generator/DocumentFormat.OpenXml.Generator.OpenXmlGenerator/Linq.P188.g.cs:96]:
```csharp
public partial class ....
{
[Obsolete("This is invalid for the namespace - use .... instead")]
public static readonly XName ext = p188 + "ext";
}
```

File: generated/DocumentFormat.OpenXml/DocumentFormat.OpenXml.Generator/DocumentFormat.OpenXml.Generator.OpenXmlGenerator/schemas_microsoft_com_office_powerpoint_2018_8_main.g.cs [hunk 2]
```diff
@@ -1188,95 +1192,6 @@ internal override void ConfigureMetadata(ElementMetadata.Builder builder)
         public override OpenXmlElement CloneNode(bool deep) => CloneImp<Comment>(deep);
     }
```
Comment (twsouthwick) [inline generated/DocumentFormat.OpenXml/DocumentFormat.OpenXml.Generator/DocumentFormat.OpenXml.Generator.OpenXmlGenerator/schemas_microsoft_com_office_powerpoint_2018_8_main.g.cs:1194]:
```csharp
namespace ....;
{
   public partial class CommentPropertiesExtension: DocumentFormat.OpenXml.Presentation.CommentPropertiesExtension
  {
  ... [ constructors ]
  }
}

```

File: src/DocumentFormat.OpenXml.Linq/Presentation/P188.cs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        /// </summary>
+        /// <remarks>
+        /// <para>As an XML element, it:</para>
+        /// <list type="bullet">
+        /// <item><description>has the following parent XML elements: <see cref="extLst" />.</description></item>
+        /// <item><description>has the following child XML elements: <see cref="P223.reactions" />, <see cref="P228.taskDetails" />.</description></item>
+        /// <item><description>corresponds to the following strongly-typed classes: CommentPropertiesExtension.</description></item>
+        /// </list>
+        /// </remarks>
+        [Obsolete("Unused property, prefer DocumentFormat.OpenXml.Linq.P.ext", true)]
```
Comment (twsouthwick) [inline src/DocumentFormat.OpenXml.Linq/Presentation/P188.cs:22]: Let's also add `[EditorBrowsable(EditorBrowsableState.Never)]` so it doesn't show up in intellisense

File: src/DocumentFormat.OpenXml/Schema/Presentation/CommentPropertiesExtension.cs [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    /// <para>When the object is serialized out as xml, it's qualified name is p:ext.</para>
+    /// </summary>
+    /// <remark>
+    /// <para>The following table lists the possible child types:</para>
+    /// <list type="bullet">
+    ///   <item><description><see cref="DocumentFormat.OpenXml.Office.PowerPoint.Y2022.M03.Main.Reactions" /> <c>&lt;p223:reactions></c></description></item>
+    ///   <item><description><see cref="DocumentFormat.OpenXml.Office.PowerPoint.Y2022.M08.Main.TaskDetails" /> <c>&lt;p228:taskDetails></c></description></item>
+    /// </list>
+    /// </remark>
+    [Obsolete("Unused type, prefer DocumentFormat.OpenXml.Presentation.CommentPropertiesExtension", true)]
```
Comment (twsouthwick) [inline src/DocumentFormat.OpenXml/Schema/Presentation/CommentPropertiesExtension.cs:21]: can you add `[EditorBrowsable(EditorBrowsableState.Never)]` as well?

**Review by twsouthwick — APPROVED** [review]
LGTM once the tests are passing and the additional attribute is added

File: src/DocumentFormat.OpenXml.Linq/Presentation/P188.cs (hunk 3 above)
Comment (tomjebo) [reply src/DocumentFormat.OpenXml.Linq/Presentation/P188.cs:22]: Definitely. Thanks.

File: src/DocumentFormat.OpenXml/Schema/Presentation/CommentPropertiesExtension.cs (hunk 4 above)
Comment (tomjebo) [reply src/DocumentFormat.OpenXml/Schema/Presentation/CommentPropertiesExtension.cs:21]: yup!

**Review by mikeebowen — APPROVED** [review]

**Review by mikeebowen — APPROVED** [review]

**Review by twsouthwick — APPROVED** [review]

**Pull request merged**